packages/network/src/connection/WebRtcEndpoint.ts

```diff
@@ -49,8 +49,7 @@ export class WebRtcEndpoint extends EventEmitter implements IWebRtcEndpoint {
     private readonly bufferThresholdHigh: number
     private readonly maxMessageSize
 
-    private statusReportTimeout?: NodeJS.Timeout
-    private statusReportIntervalMs: number
+    private statusReportInterval?: NodeJS.Timeout
```

Comment:
The `statusReportInterval` name is maybe misleading, as this variable contains the timer id of `setInterval` call, not the interval length.  Could be e.g.`statusReportTimer`.